canonical-web-and-design/ubuntu.com
# [WIP] Generate microcerts table with dummy data

## Done

- Reactify the Microcerts table and generate rows using dummy data

## QA

- Go to http://localhost:8001/cube/microcerts
- Verify that the Microcerts table appears with 5 rows and that the appropriate button appear for each status (Enrolled, Not Enrolled, Passed, Failed, In Progress)
  - See [wireframe](https://miro.com/app/board/o9J_l1zDzLY=/)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/251

## Screenshots

![microcerts-react](https://user-images.githubusercontent.com/995051/135488918-9443b3ad-24d6-4aa8-83b5-d9c2af43bdf5.png)
